src/applications/vaos/sass/vaos.scss

```diff
@@ -95,4 +95,14 @@ va-radio::part(legend) {
 .schemaform-required-span,
 body {
   font-size: 1.06rem; // same as 16.96px
+}
+
+.vaos__dynamic-font-size--h2 {
+  font-size: var(--font-size-h2);
```

Comment:
`--font-size-h2` is defined in `~@department-of-veterans-affairs/css-library/dist/tokens/scss/variables` as `1.875 rem`. This matches what's set by `core.css` but the only risk is that they might not match in the future. Do we know if `scss/variables` and `core.css` are always kept in sync?

Reply:
I don't know.